cmd/state/internal/cmdtree/prepare.go

```diff
@@ -0,0 +1,25 @@
+package cmdtree
+
+import (
+	"github.com/ActiveState/cli/internal/captain"
+	"github.com/ActiveState/cli/internal/primer"
+	"github.com/ActiveState/cli/internal/runners/prepare"
+)
+
+func newPrepareCommand(prime *primer.Values) *captain.Command {
+	runner := prepare.New(prime)
+
+	cmd := captain.NewCommand(
+		"_prepare",
+		"for programmatic use only",
```

Comment:
```suggestion
		locale.Tl("Prepare environment for use with the State Tool."),
```

Reply:
I added this exact message, but provided a l10n key according to `Tl` usage.